buildSrc/src/main/groovy/org/elasticsearch/gradle/test/ClusterFormationTasks.groovy

```diff
@@ -391,6 +391,17 @@ class ClusterFormationTasks {
         return configureExecTask(name, project, setup, node, args)
     }
 
+    /** Surround strings that contains a comma with double quotes **/
+    private static String escapeComma(Object o) {
+        if (o instanceof String) {
+            String s = (String)o
```

Comment:
Good point, I'm not yet used to see these kind of things at the first sight.

I created https://github.com/elastic/elasticsearch/pull/17502 for that, would be great if you can have a look